src/utils/testSupport/TestingUtils.ts

```diff
@@ -285,6 +296,24 @@ export class TestInput {
 
         return dummyControlInput(userEvent);
     }
+
+    public static canFulfillIntentRequest(
```

Comment:
Why is this static method defined? I see no test cases or utils referencing this method ?

Reply:
Because i used it in the `dj-song-request` component unit test: 
https://code.amazon.com/reviews/CR-55630822/revisions/1#/diff#AskIMESongRequestCustomSkillPOC_test_CustomRootControl_spec_ts

Reply:
Ahh, can you add similar test case in the framework as well, for 3p developers who explore the codebase to understand better? Also improves code coverage too

Reply:
Ok, will add.